bin/ci/script.sh

```diff
@@ -11,6 +11,12 @@
 
 cd "$(dirname "$0")" || exit; source _includes.sh
 
+# Switch to PHP8.
+if [[ "$PHP8_JOB" == TRUE && "$JENKINS_HOME" ]]; then
+  update-alternatives --install /usr/local/bin/php php /usr/bin/php8.0 80
```

Comment:
This isn't necessary, because the same code you added in `bin/ci/install.sh` has already been run when this file is executed.